controllers/productController.js

```diff
@@ -5,7 +5,7 @@ const getProduct = catchAsync(async (req, res) => {
   const { productId } = req.params;
   const productDetail = await productService.getProduct(productId);
 
-  return res.status(200).json({ productDetail });
+  return res.status(200).json(productDetail);
```

Comment:
객체를 제거한 이유가 있을까요??
이렇게 보내면 key가 없는 상태로 배열이 전송 될 것 같은데, 
데이터를 보낼 때는 key 같이 보내서 데이터를 특정 지을 수 있게 해주는 것이 좋습니다. 
다시 추가해 주세요.
```suggestion
  return res.status(200).json({productDetail});
```